src/main/java/calculator/Number.java

```diff
@@ -0,0 +1,48 @@
+package calculator;
+
+import java.util.Objects;
+
+public class Number {
+    private static final String NEGATIVE_ERROR_MESSAGE = "양수를 입력해야 합니다";
+    private static final String ZERO = "0";
+
+    private final int number;
+
+    public Number(String numberString) {
+        if (isNullOrEmpty(numberString)) {
+            numberString = ZERO;
+        }
+
+        int numberParsed = Integer.parseInt(numberString);
+        if (isNegative(numberParsed)) {
+            throw new RuntimeException(NEGATIVE_ERROR_MESSAGE);
+        }
+
+        this.number = numberParsed;
+    }
+
+    private boolean isNegative(int numberParsed) {
+        return numberParsed < 0;
+    }
+
+    private boolean isNullOrEmpty(String numberString) {
+        return numberString == null || "".equals(numberString);
```

Comment:
String 클래스의 isEmpty() 메서드를 활용해도 좋을 것 같아요 :)

```suggestion
        return numberString == null || numberString.isEmpty();
```

Reply:
감사합니다!ㅎㅎ